docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,51 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` | user | edit a person | change details of a person easily
+|`* *` | user | import persons | share with other users and migrate from other systems
+|`* *` | user | export persons | share with other users and migrate from other systems
+|`* * *` | user | tag a person | store some notes about certain people
+|`* * *` | user | categorize persons | easily organize contacts and later on find wrt group
+|`* *` | user with many persons in the address book | apply filter: category, tag, name | easily view and process only certain group
+|`*` | user with many persons in the address book | list categories with statistics | see how my contacts are grouped
+|`* *` | user | use password to get access to my contacts | protect my data
+|`* *` | user | clear the address book | start anew
+|`* *` | user | see suggestions while typing | type commands faster
+|`* *` | impatient user | get the previously entered commands into the input field | type commands faster
+|`* * *` | user with many persons in the address book | find all the persons that satisfy part of the name (eg. Jo*) | search without remembering all the details
+|`* * *` | user | get closest result if my input doesn't match any name | I get results even if I misspell
+|`* * *` | user | add a picture to a person | recall easily who the person is
+|`* *` | new user | use a tutorial | learn the features of the app quickly
+|`* *` | user | remove/categorize/tag all the persons selected by a filter | process multiple contacts quickly
+|`* * *` | user | add a social media link to a person | get more info about a person in one place (in the App)
+|`* *` | impatient user | use hotkeys | navigate faster
+|`* * *` | user | be warned before execution of a hazardous command | prevent from disastrous mistakes
+|`* * *` | user | change the theme of the GUI (dark/bright) | prevent eye strain or simply customize
+|`* *` | user | add new fields to all the persons | customize the App
+|`* * *` | user | validate persons details when adding
+|`* * *` | user | input more than 1 phone number | can store home + office + mobile numbers easily
+|`* * *` | user | search for people via different fields (address, phone number etc.) | easily view people of same area of residence
+|`* * *` | user | allow mass deletion | easy cleanup
+|`* * *` | user | have a favourites tag that is always easily accessible | easily access most frequent contacts
+|`* *` | user | sort by frequency of access | easily access most frequent contacts
+|`* *` | forgetful user | have a reminder which reminds me to call someone at certain times | call my contacts if needed
+|`* *` | busy user | save appointment with people in my addressbook and send them a message about the appointment | set up appointment with my contacts easily
+|`* *` | stealthy users | have hidden contacts which can only be shown upon typing a secret phrase | hide my other girlfriends from my girlfriend
```

Comment:
i think mentioning here a password would be more proper, but then we would like to have two passwords, right? Isnt that a bit weird?

Reply:
perhaps just have private contacts in addition to private contact details

Reply:
I think the case of 2 passwords is actually fine. The passwords can even be the same, so the hidden contacts feature prevents people using your addressbook from accessing them, much like how you can put a password lock on some apps of your own smartphone to prevent your friends from seeing things when you lend them your phone